package/MDAnalysis/coordinates/base.py

```diff
@@ -1205,6 +1241,13 @@ def _read_frame(self, frame):
         #                                  ts._unitcell, 1)
         # return ts
 
+    def _goto_frame(self, frame):
+        """Move to *frame*, updating ts with trajectory and auxiliary data."""
+        ts = self._read_frame(frame)
+        for aux in self.aux_list:
```

Comment:
Read frame is redefined by the readers. Dealing with the auxiliary fields should not be their responsibility.